docs/modules/ROOT/pages/advanced-config.adoc

```diff
@@ -74,9 +74,32 @@ spec:
 [TIP]
 .Passing environment variables
 ====
-If you want to pass environment variables to the `backupCommand` you'll have to wrap them in a shell. Example:
+If you want to pass environment variables to the `backupCommand` you'll have to wrap them in a shell. In above example that would look like this:
```

Comment:
I am not sure to which example you are referring to?

Reply:
This part is right beneath the prebackup pod description. So the example above is the backuppod example :) Makes more sense when viewing the whole file.